agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp

```diff
@@ -1760,46 +1760,6 @@ bool monitor_thread::hal_event_handler(bwl::base_wlan_hal::hal_event_ptr_t event
 
     } break;
 
-    case Event::RRM_Link_Measurement_Response: {
```

Comment:
@arnout is this message really deprecated or should it remain and its information is translated to a non-VS message?